torch/library.py

```diff
@@ -139,6 +139,58 @@ def _register_fake(self, op_name, fn, _stacklevel=1):
         handle = entry.abstract_impl.register(func_to_register, source)
         self._registration_handles.append(handle)
 
+    def _impl_with_aoti_compile(self, op_name, op_overload_name='', dispatch_key='', fallback_fn=None):
```

Comment:
To match `impl`, we shouldn't have an op_overload_name -- all the info is included in op_name.